gradio/queueing.py

```diff
@@ -35,7 +37,7 @@ def __init__(
         self._id = f"{self.session_hash}_{self.fn_index}"
         self.data: PredictBody | None = None
         self.lost_connection_time: float | None = None
-        self.token: str | None = None
+        self.username: Optional[str] = None
```

Comment:
For consistency with the rest of the codebase:
```suggestion
        self.username: str | None = None
```